### ISSUE TITLE
Footer Redesign

### ISSUE THREAD

**Issue by sosiology** [visualize-admin/electricity-prices-switzerland#60]
Elcom intends to add more links to the footer (amoungst other a link to a dedicated Elcom/Visualize page). At the moment the links have just been “placed” at the bottom of the page, however there may be an opportunity to redesign and optimize the footer, making it more scalable in future. Possible something like for the [Covid Dashboard](https://www.covid19.admin.ch/de/overview) or https://energiedashboard.admin.ch/dashboard. 

We will prepare a design suggestion/mockup to present to the client for approval.

**Comment by AnninaWalker** [comment]
@sosiology Here a designproposal for the footer. I basically took over the same design as for the Covid Dashboard and added some potential links.
I've changed the title from "Daten herunterladen" to "Daten herunterladen / visualisieren" and added a link that could lead to a potential Elcom / Visualize page at the bottom.

![Image](https://user-images.githubusercontent.com/59613691/214244610-5cb7176e-15e0-4ee0-a325-2c97cdddeae1.jpg)

**Comment by sosiology** [comment]
@ptbrowne will implement the new footer and share function behind a flag to share with the client for review.

**Comment by ptbrowne** [comment]
@sosiology Do we have the links for all the external links ?

        Berechnungsgrundlagen
        Energiesparen
        Versorgungslage
        Stromversorgung
        Gasversorgung

**Comment by sosiology** [comment]
@ptbrowne i believe these links were suggestions from Annina. For now I would suggest adding the following for now: 

**Weiterführende Informationen**

-  **Berechnungsgrundlagen** - which opens the modal with further information on how the prices are calaculated 
- **Showcase auf opendata.swiss** - which points here: https://opendata.swiss/de/showcase/electricity-tariffs 
- **Elcom Website** - which points to https://www.elcom.admin.ch/elcom/de/home.html 

**Daten Herunterladen**
- **Rohdaten 2022 (CSV-Datei)** which initiates the download for the data file 
- **Elcom Daten visualisieren** - which points to the Elcom data sets on Visualize:  https://visualize.admin.ch/de/browse?includeDrafts=true&search=Elcom&order=SCORE&dataSource=Prod


Once we have a version of the footer ready for review on TEST (or REF) I can ask the client which links to include, respectively to check the ones we added and let us know if those make sense for the time being.

**Comment by ptbrowne** [comment]
Thanks Solange, in the end I saw some of the links that Annina was suggesting were in the footer of the Energy dashboard, I took the links from there. For visualize, I had taken the homepage but it's better with the search that you mentioned, thanks for the suggestion.

**Comment by sosiology** [comment]
Thanks @ptbrowne for preparing this. My Feedback below:

**Page width**
Personally, I am not a fan of limiting the width. In comparison to the covid dashboard for example, the map is not shown in a card, so I personally think for Elcom it is nice to have the ability to use all the available screen space to explore the map. When the browser screen is minimized (e.g. half screen) it also looks strange as there is no clear boundary.
<img width="1765" alt="image" src="https://github.com/visualize-admin/electricity-prices-switzerland/assets/29289679/ddb8adf6-9eb4-4a8f-b3cb-25faf48aa601">

I understand that this makes it easier to scroll and is better suited for the new footer but would prefer to not implement this change now if the motivation is purely to improve the ergonomic experience, as this is not something that has ever been discussed with the client, requested to improve or received as user feedback. If there are other considerations I am missing - happy to further discuss or review it with a designer. 

**Share / Download Buttons**
I would remove the background color - I think the white background looks a bit strange
<img width="356" alt="image" src="https://github.com/visualize-admin/electricity-prices-switzerland/assets/29289679/b6764eb2-ee2e-4ef7-8650-8ea8c6e434dd">

**Background Color Behind Map**
I assume you made it a lighter grey to better differentiate it from the footer?

**Links**
The links in the footer all point to BFE. I think this should before sharing it with the client for review. AFAIK BFE is not the same as Elcom.

Also as the links lead to other pages / domains, i think they should be opened in a new tab. 

My Proposal: 
Berechnungsgrundlagen -> OK
Energiesparen -> OK
Versorgungslage -> change to Versorgungssicherheit and point to https://www.elcom.admin.ch/elcom/de/home/themen/versorgungssicherheit.html
Stromversorgung -> Remove (as it points to the BFE page)
Gasversorgung -> Remove (as it points to the BFE page)

Daten als .csv -> OK
Daten auf opendata.swiss -> should point to elcom data (not BFE), which is currently not available on opendata swiss. So for the timebeing I would point it to here: https://opendata.swiss/de/showcase/electricity-tariffs as this page mentions that the data is not yet available.
Datenvisualisierungen Erstellen -> as you commented above - if we can directly show the elcom data sets (they are currently drafts) I think the client would appreciate that.

**Comment by ptbrowne** [comment]
I had merged a bit early, I've implemented the changes and it is visible here : https://elcom-strompreis-dev.interactivethings.io/.

**Page width**

OK, let's postpone. I personally feel like the user experience of the homepage is not very good. It's true that we haven't heard that from the client or from user testing but I would not count it as something as the client is not super proactive. Let's have the opinion of a designer.

**Share / Download Buttons**

This is to increase contrast with the map below and is part of the design. I just noticed that this was the same kind of background that we apply to the legend and used the same background and same border radius. I would be in favor of keeping it.

Without it, I think it is much worse:

<img width="600" alt="image" src="https://github.com/visualize-admin/electricity-prices-switzerland/assets/465582/8e9f9fa3-75c2-4cfa-80db-7e2ca98db6ed">

**Links**

Great feedback, thanks. I've implemented the changes.
>  as you commented above - if we can directly show the elcom data sets (they are currently drafts) I think the client would appreciate that.

Not sure if I need to do something here, the link correctly preselect the draft checkbox, so it seems OK right ?
